app/javascript/app/components/vulnerability-graph/vulnerability-graph-component.jsx

```diff
@@ -69,7 +69,7 @@ class VulnerabilityGraph extends PureComponent {
         </div>
       </div>
     ) : (
-      'No data'
+      <div className={styles.noData}>No data</div>
```

Comment:
We already have a component called `NoContent` for that kind of this, could you please use it?